owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt

```diff
@@ -184,7 +182,7 @@ class FileDisplayActivity : FileActivity(), FileFragment.ContainerActivity, OnEn
 
         /// Load of saved instance state
         if (savedInstanceState != null) {
-            Log.d(TAG, savedInstanceState.toString())
+            Log_OC.d(savedInstanceState.toString())
```

Comment:
Please switch direct to `Timber` and **delete** `Log_OC` complete, then I can get rid of the need of https://github.com/JakeWharton/timber/pull/371 and can use common upstream version instead of special version which support [delegation](https://github.com/hannesa2/Logcat/blob/5cba849239a0d71ef21f69d91c986c875c16b580/LogcatLib/build.gradle#L28-L29)

Reply:
It's not a hard requirement for this timber-only task, but I start to remove it https://github.com/hannesa2/Logcat/pull/31

Reply:
I created a new issue yesterday related with Logs (https://github.com/owncloud/android/issues/2757) We can discuss it there. This PR is about to clean a little bit. The idea of this commit (https://github.com/owncloud/android/pull/2750/commits/deba77cba80b411ed863335c0b077d4fb1520230) is to standardize Log_OC as log solution. This will help if we decide to replace Log_OC completely too.

Reply:
> Please switch direct to Timber and delete Log_OC complete, then I can get rid of the need of JakeWharton/timber#371 and can use common upstream version instead of special version which support delegation

@abelgardep will do it but in a separate PR, this one has too many changes already